TeachingRecordSystem/src/TeachingRecordSystem.SupportUi/Pages/Persons/PersonDetail/Index.cshtml

```diff
@@ -34,18 +34,59 @@
 }
 @await Html.PartialAsync("_PersonDetail", personDetailModel)
 
-@{
-    if (Model.PersonProfessionalStatus is not null)
-    {
-        var personStatusDetailModel = new PersonProfessionalStatusDetailViewModel()
-        {
-            EytsDate = Model.PersonProfessionalStatus.EytsDate,
-            HasEyps = Model.PersonProfessionalStatus.HasEyps,
-            InductionStatusInfo = InductionStatusRegistry.All.Single(s => s.Value == Model.PersonProfessionalStatus.InductionStatus),
-            PqtsDate = Model.PersonProfessionalStatus.PqtsDate,
-            QtsDate = Model.PersonProfessionalStatus.QtsDate,
-            QtlsStatus = Model.PersonProfessionalStatus.QtlsStatus
-        };
-        @await Html.PartialAsync("_PersonProfessionalStatusDetail", personStatusDetailModel)
-    }
+@if (Model.PersonProfessionalStatus is not null)
+{
+    <govuk-summary-card>
+        <govuk-summary-card-title data-testid="professional-status-details">Professional status details</govuk-summary-card-title>
+        <govuk-summary-list>
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Qualified teacher status (QTS)</govuk-summary-list-row-key>
+                <govuk-summary-list-row-value>@(Model.PersonProfessionalStatus.QtsDate != null ? "Holds" : "No")</govuk-summary-list-row-value>
+            </govuk-summary-list-row>
+            @if (Model.PersonProfessionalStatus.QtsDate is not null)
+            {
+                <govuk-summary-list-row>
+                    <govuk-summary-list-row-key>QTS held since</govuk-summary-list-row-key>
+                    <govuk-summary-list-row-value>@Model.PersonProfessionalStatus.QtsDate?.ToString(UiDefaults.DateOnlyDisplayFormat)</govuk-summary-list-row-value>
+                </govuk-summary-list-row>
+            }
+            <govuk-summary-list-row>
+                <govuk-summary-list-row-key>Qualified Teacher Learning and Skills status (QTLS)</govuk-summary-list-row-key>
```

Comment:
sentence case like the other labels?

Reply:
> sentence case like the other labels?

It's correct in this case (no pun intended).